src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div
-		class="kv-tailwind tw-w-[336px]"
+		class="kv-tailwind tw-min-w-[230px] tw-max-w-[374px]"
```

Comment:
To keep these classes from being added to the global stylesheet, can you please move these min/max widths to inline styles